config.js

```diff
@@ -74,8 +74,8 @@ var config = {
         // callStatsThreshold: 5 // enable callstats for 5% of the users.
     },
 
-    // Enables reactions feature.
-    // enableReactions: false,
+    // Disables the reactions feature.
+    // disableReactions: true,
```

Comment:
Reactions are already available in a stable release, if you change this you'll break them. Just change the code to `undefined` is treated as `true`.

Reply:
@saghul what is your concern here, what will break?

So the change is now, that if you are missing disableReactions in config js this will default to reactions disabled ... this is not enabling by default? All users need to go and edit the config.js to add disableReactions: false in order to enable them? 

Let's drop the case that someone had used the latest stable and had disabled reactions with `enableReactions: false` ... as the stable was out for 10 days. Maybe at end of this week or early next week, we will update stable. There are numerous FF fixes so we want a new stable ...